doc/api.rst

```diff
@@ -72,29 +72,43 @@ options as the constructor second argument::
 
 The following options are available:
 
-* ``debug``: When set to ``true``, the generated templates have a
+* ``debug`` *boolean*
+
+  When set to ``true``, the generated templates have a
   ``__toString()`` method that you can use to display the generated nodes
   (default to ``false``).
 
-* ``charset``: The charset used by the templates (default to ``utf-8``).
+* ``charset`` *string (default to ``utf-8``)*
+
+  The charset used by the templates.
+
+* ``base_template_class`` *(default to ``Twig_Template``)*
+
+  The base template class to use for generated
+  templates.
 
-* ``base_template_class``: The base template class to use for generated
-  templates (default to ``Twig_Template``).
+* ``cache`` *boolean*
 
-* ``cache``: An absolute path where to store the compiled templates, or
+  An absolute path where to store the compiled templates, or
   ``false`` to disable caching (which is the default).
 
-* ``auto_reload``: When developing with Twig, it's useful to recompile the
+* ``auto_reload`` *boolean*
+
+  When developing with Twig, it's useful to recompile the
   template whenever the source code changes. If you don't provide a value for
   the ``auto_reload`` option, it will be determined automatically based on the
   ``debug`` value.
 
-* ``strict_variables``: If set to ``false``, Twig will silently ignore invalid
+* ``strict_variables`` *boolean*
+
+  If set to ``false``, Twig will silently ignore invalid
   variables (variables and or attributes/methods that do not exist) and
   replace them with a ``null`` value. When set to ``true``, Twig throws an
   exception instead (default to ``false``).
 
-* ``autoescape``: If set to ``true``, HTML auto-escaping will be enabled by
+* ``autoescape`` *boolean*
```

Comment:
one of the supported escaping strategies (``true`` is an alias for ``html`` for BC reasons)